google/auth/external_account.py

```diff
@@ -389,6 +391,48 @@ def with_token_uri(self, token_uri):
         kwargs.update(token_url=token_uri)
         return self.__class__(**kwargs)
 
+    def introspect_token(self, request):
+        """Introspection of token info.
```

Comment:
"Introspection of token info" doesn't quite make sense -- let's replace with something else

Reply:
Changed.